hack/scripts/process-unit-test-results.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+
+# Copyright 2023 VMware, Inc. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+# This script accepts a file directory path which has ginkgo generated test reports json files as an argument and processes the json files and generates a github flavored markdown table of Test results summary
+
+# Read the data from the file
+echo "loading data from $1"
+file_data=$(cat $1)
+
+echo "done loading data from $1"
+
+# Function to remove ASCII escape sequences
+remove_escape_sequences_a() {
+    echo "$1" | sed -E 's/\x1B\[[0-9;]*[a-zA-Z]//g'
+}
+
+# Function to remove ASCII escape sequences from a string
+remove_escape_sequences() {
+    echo -e "$1" | sed -E 's/\x1B\[([0-9]{1,2}(;[0-9]{1,2})*)?[mGK]//g'
+}
+
+# Function to extract numeric value from a line
+extract_numeric_value() {
+    echo "$1" | grep -oE '[0-9]+'
+}
+
+# Print the table header
+echo "| :memo: Test Suite And Path | Status | Total Tests | Passed | Failed | Pending | Skipped | Coverage"
```

Comment:
I don't think Status column is of much use here ?. Can we just add colors like red to Failures and maybe orange pending blue skipped something like this.